src/JoomlaBrowser.php

```diff
@@ -1099,26 +1042,195 @@ public function createCategory($title, $extension = '')
 		}
-
 		$this->amOnPage('administrator/index.php?option=com_categories' . $extension);
-
 		$this->waitForElement(['class' => 'page-title']);
 		$this->checkForPhpNoticesOrWarnings();
-
 		$this->debug('Click new category button');
 		$this->click($this->locator->adminToolbarButtonNew);
-
 		$this->waitForElement(['class' => 'page-title']);
-
 		$this->checkForPhpNoticesOrWarnings();
 		$this->fillField(['id' => 'jform_title'], $title);
-
 		$this->debug('Click new category apply button');
 		$this->click($this->locator->adminToolbarButtonApply);
-
 		$this->debug('see a success message after saving the category');
-
 		$this->see('Category saved', ['id' => 'system-message-container']);
 		$this->checkForPhpNoticesOrWarnings();
 	}
+
+	/**
+	 * Site Offline
+	 *
+	 * @return  void
+	 *
+	 * @since   3.7.5
+	 */
+	public function setSiteOffline($enable == true)
+	{
+		$this->debug('I set the site to offline');
+		$this->doAdministratorLogin();
+		$this->amOnPage('/administrator/index.php?option=com_config');
+		if($enable == true){
+			$this->click(['id' => 'jform_offline1']);
+		}else{
+			$this->click(['id' => 'jform_offline0']);
+		}
+		$this->click("Save");
+		$this->see('Configuration saved', ['id' => 'system-message-container']);
+	}
+
+	/**
+	 * Search Engine Optimization
+	 *
+	 * @param   \AcceptanceTester   $I  Acceptance Tester
+	 *
+	 * @return void
+	 */
+	public function setSiteSearchEngineFriendly($enable == true)
+	{
+		$this->debug('I set the Search engine optimisation to Yes');
+		$this->doAdministratorLogin();
+		$this->amOnPage('/administrator/index.php?option=com_config');
+		// If-else statement
+		if ($enable == true){
+			$this->click(['id' => 'jform_sef1']);
+			//Select Option 'After'
+			$this->selectOption(['id' => 'jform_sitename_pagetitles'],'After');
+		}
+		else
+		{
+			$this->click(['id' => 'jform_sef0']);
+		}
+		$this->click("Save");
+		$this->see('Configuration saved', ['id' => 'system-message-container']);
+	}
+
+	/**
+     * Create Module
+     *
+     * @param   \AcceptanceTester $I          Acceptance Tester
+     * @param   string            $scenario   Scenario
+     *
+     * @return void
+     */
+    public function createModule($moduleType, $moduleName, $category)
+    {
+        $this->doAdministratorLogin();
+        $this->amOnPage(Locators::$moduleUrl);
+        // New Module
+        $this->click('New');
+        // Select Module Type
+        $this->see('Select a Module Type');
+        // Module Type
+        $this->click(['link' => $moduleType]);
+        // Title
+        $this->fillField(Locators::$moduleTitle, $moduleName);
+        /**
+         * Set Position
+         * setModulePosition not working. Tried both
+         * $I->setModulePosition($moduleName,'Main Top[main-top]'); and
+         * $I->setModulePosition($moduleName,'Main Top');
+         */
+        $this->click('Type or Select a Position');
+        $this->fillField(Locators::$modulePosition,'Main Top');
```

Comment:
leave a space before the second parameter
`$this->fillField(Locators::$modulePosition, 'Main Top');`